src/Form/PathautoPatternDeleteForm.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/**
+ * @file
+ * Contains Drupal\pathauto\Form\PathautoPatternDeleteForm.
+ */
+
+namespace Drupal\pathauto\Form;
+
+use Drupal\Core\Entity\EntityConfirmFormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Url;
+
+/**
+ * Builds the form to delete Pathauto pattern entities.
+ */
+class PathautoPatternDeleteForm extends EntityConfirmFormBase {
```

Comment:
Yes, pretty sure that Drupal\Core\Entity\EntityDeleteForm should just work for this.